action.yml

```diff
@@ -11,10 +11,13 @@ inputs:
   triggered_key:
     description: 'A uuid specific for this job added by user'
     required: false
+  extra-arg:
```

Comment:
Maybe instead of the hard coded "extra_arg" make it a json of extra kwargs (you can even rename it to kwargs), and just expand it in the parameters to circle ci, that way it will be more generic, and you'll be able to pass multiple args with whatever name you want

Maybe we can even remove the existing triggered_key then